README.md

```
@@ -1,18 +1,65 @@
+<p>Big thanks to the developers and strategists who have helped build this game!</p>
 
 <a href="https://github.com/Rakshitg600/notakto-website/graphs/contributors">
-  <img src="https://contrib.rocks/image?repo=Rakshitg600/notakto-website" alt="Contributors to Notakto GitHub repository" />
+  <img src="https://contrib.rocks/image?repo=Rakshitg600/notakto-website&columns=10&max=20&anon=true" alt="Contributors" style="border-radius: 10px;" />
 </a>
 
+<br />
+<br />
+
+<p>
+  <img src="https://img.shields.io/badge/License-Unlicense-lightgrey?style=flat-square" />
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)